core/src/lib.rs

```diff
@@ -76,14 +76,13 @@ impl CorePlugin {
 
 impl Plugin for CorePlugin {
     fn build(&self, app: &mut AppBuilder) {
-        app.resources_mut().get_or_insert_with(Gravity::default);
-
-        app.register_type::<Body>()
+        app.init_resource::<Gravity>()
+            .register_type::<Body>()
             .register_type::<BodyType>()
             .register_type::<PhysicMaterial>()
             .register_type::<Velocity>()
             .register_type::<RotationConstraints>()
-            .add_stage_after(bevy::app::stage::UPDATE, crate::stage::ROOT, {
+            .add_stage_before(CoreStage::Update, crate::stage::ROOT, {
```

Comment:
Was this change intentional?

Reply:
This was a mistake. Thanks for the hint ;-)